libraries/recorder.js

```diff
@@ -1,70 +1,190 @@
-var fs = require('fs');
+const fs = require('fs');
```

Comment:
It would be great for newcomers to have brief comments what main methods do.

Reply:
Good point, thanks!